src/Heating/Heater.h

```diff
@@ -200,6 +201,7 @@ class Heater INHERIT_OBJECT_MODEL
 	int sensorNumber;								// the sensor number used by this heater
 	float activeTemperature;						// the required active temperature
 	float standbyTemperature;						// the required standby temperature
+	float lastActiveTemperature;					// the last non-zero temperature
```

Comment:
This variable is not initialized anywhere, please fix it

Reply:
Fixed